app/components/FormFields/validate.js

```diff
@@ -74,10 +74,14 @@ const requiredForObjectField = value => {
   return !val || (val && !val.value) ? messages.requiredField : undefined;
 };
 
+const tagNameHasNotExist = suggestedTags => value =>
```

Comment:
All validators in this js file are reusable. They should have generic names that can be reused in other places.

Reply:
There is already `valueHasNotBeInList` that can be used. No need to create new validator.